venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,340 @@
+    original_arr_sub_id = None
+    #submission_output_dict['acl_commitment_forum'] = acl_commitment_note.forum
+    try:
+        original_arr_sub = client.get_note(arr_submission_forum)
+        if(original_arr_sub.original):
+            original_arr_sub_id = original_arr_sub.original
+        else: 
+            original_arr_sub_id = original_arr_sub.id
+    except:
+        print(f"Note {arr_submission_forum} does not exist")
+    if original_arr_sub_id and ((client.get_note(original_arr_sub_id)).invitation.startswith(tuple(eligible_arr_invitations))):
+        original_arr_sub = client.get_note(original_arr_sub_id)
+        submission_output_dict[acl_commitment_note.forum]['arr_submission_forum'] = original_arr_sub.forum
+        if acl_commitment_note.signatures[0] in original_arr_sub.content['authorids']:
+            # Create new note to submit to ACL 
+            acl_sub = openreview.Note(
+                invitation="aclweb.org/ACL/2022/Conference/-/Submission",
+                readers = [
+                    "aclweb.org/ACL/2022/Conference"
+                    ], 
+                writers = [
+                    "aclweb.org/ACL/2022/Conference"
+                    ],
+                signatures = [
+                    "aclweb.org/ACL/2022/Conference"
+                    ],
+                content = {
+                    "paper_link": note.content["paper_link"],
+                    "paper_type":note.content["paper_type"],
+                    "track":note.content["track"],
+                    "comments_to_the_senior_area_chairs":note.content.get("comments_to_the_senior_area_chairs"),
+                    "authorids":original_arr_sub.content["authorids"],
+                    "authors": original_arr_sub.content["authors"],
+                    "title":original_arr_sub.content["title"],
+                    "abstract":original_arr_sub.content["abstract"],
+                    "data":original_arr_sub.content.get("data"),
+                    "software":original_arr_sub.content.get("software"),
+                    "pdf":original_arr_sub.content.get("pdf"),
+                    "acl_preprint": note.content.get("ACL_preprint"),
+                    "existing_preprints": note.content.get("existing_preprints"),
+                    "authorship": note.content.get("authorship"),
+                    "paper_version": note.content.get("paper version"),
+                    "anonymity_period": note.content.get("anonymity period"),
+                    "commitment_note": f"https://openreview.net/forum?id={acl_commitment_note.forum}"
+                }  
+            )
+            acl_submitted = client.post_note(acl_sub)
+            if(acl_submitted):
+                submission_output_dict['was_migrated'] = True
```

Comment:
should this be something like `submission_output_dict[acl_commitment_note.forum]['was_migrated'] = True`?

Reply:
Yes! Changed this